### PR TITLE
introduced DebugUtils plugin

### PR DESCRIPTION
**Pull request by MiranDMC**
new opcode 0DD5 (get_platform)
new opcode 00CC (breakpoint)
new opcode 00CD (trace)
new opcode 00CE (log_to_file)
support of opcodes 0662, 0663 and 0664
more error and warning messages
updated opcodes handling

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,4 +38,7 @@ build/
 Debug/*
 Release/*
 ipch/
-.vs/
\ No newline at end of file
+.vs/
+*/.output
+*.zip
+/.output/*/*.lib
```
Comment (x87) [inline .gitignore:44]: I think all .output folder is now ignored (see line 2)

Comment (MiranDMC) [reply .gitignore:44]: Respite all this ignores I still see .lib files when committing ;/

Comment (x87) [reply .gitignore:44]: .lib files in output folder? can you show a screenshot of git/whatever you use?

Comment (MiranDMC) [reply .gitignore:44]: obraz

**Review by x87 — APPROVED** [review]

**Pull request merged**